frontend/css/playlists.less

```diff
@@ -277,3 +278,25 @@
     box-shadow: 0 0 0 2px rgba(53, 48, 112, 0.5);
   }
 }
+
+.wikipedia-extract {
+  .playlist-description {
+    overflow: hidden;
+    position: relative;
+    line-height: 1.2em;
+    max-height: @playlistDescriptionLines * 1.2em;
+    text-align: justify;
+    padding-right: 2em;
+    display: -webkit-box;
+    line-clamp: @playlistDescriptionLines;
+    margin-top: 0.5rem;
+    -webkit-line-clamp: @playlistDescriptionLines;
+    -webkit-box-orient: vertical;
+  }
```

Comment:
For what happens after clicking the show-more button, you probably want a `&.expanded` css definition that removes the line-clamp and overflow styles.
Either that or the other way around, having expanded be the standard style and use `&:not(.expanded)` to set the line-clamping and overflow.